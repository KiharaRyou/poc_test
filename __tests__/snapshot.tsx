import './matchMedia.mock';
import { render } from '@testing-library/react'
import ResetPasswordPage from '@/pages/reset_password'

it('render reset password page', () => {
  const { container } = render(<ResetPasswordPage />)
  expect(container).toMatchSnapshot()
})
