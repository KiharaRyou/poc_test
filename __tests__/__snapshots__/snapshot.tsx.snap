// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render reset password page 1`] = `
<div>
  <div
    class="ant-row css-dev-only-do-not-override-txh9fw"
  >
    <div
      class="ant-col container css-dev-only-do-not-override-txh9fw"
    >
      <span>
        Reset Passowrd
      </span>
      <span
        aria-label="question-circle"
        class="anticon anticon-question-circle"
        role="img"
      >
        <svg
          aria-hidden="true"
          data-icon="question-circle"
          fill="currentColor"
          focusable="false"
          height="1em"
          viewBox="64 64 896 896"
          width="1em"
        >
          <path
            d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
          />
          <path
            d="M623.6 316.7C593.6 290.4 554 276 512 276s-81.6 14.5-111.6 40.7C369.2 344 352 380.7 352 420v7.6c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8V420c0-44.1 43.1-80 96-80s96 35.9 96 80c0 31.1-22 59.6-56.1 72.7-21.2 8.1-39.2 22.3-52.1 40.9-13.1 19-19.9 41.8-19.9 64.9V620c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8v-22.7a48.3 48.3 0 0130.9-44.8c59-22.7 97.1-74.7 97.1-132.5.1-39.3-17.1-76-48.3-103.3zM472 732a40 40 0 1080 0 40 40 0 10-80 0z"
          />
        </svg>
      </span>
      <div
        class="ant-divider css-dev-only-do-not-override-txh9fw ant-divider-horizontal"
        role="separator"
      />
      <form
        class="ant-form ant-form-horizontal css-dev-only-do-not-override-txh9fw"
        id="resetPassword"
        style="width: 400px;"
      >
        <div
          class="ant-form-item css-dev-only-do-not-override-txh9fw"
        >
          <div
            class="ant-row ant-form-item-row css-dev-only-do-not-override-txh9fw"
          >
            <div
              class="ant-col ant-form-item-label ant-col-xs-24 ant-col-sm-12 css-dev-only-do-not-override-txh9fw"
            >
              <label
                class="ant-form-item-required"
                for="resetPassword_password"
                title="New Password"
              >
                New Password
              </label>
            </div>
            <div
              class="ant-col ant-form-item-control ant-col-xs-24 ant-col-sm-16 css-dev-only-do-not-override-txh9fw"
            >
              <div
                class="ant-form-item-control-input"
              >
                <div
                  class="ant-form-item-control-input-content"
                >
                  <span
                    class="ant-input-affix-wrapper ant-input-password ant-input-affix-wrapper-has-feedback css-dev-only-do-not-override-txh9fw"
                  >
                    <input
                      aria-required="true"
                      class="ant-input css-dev-only-do-not-override-txh9fw"
                      data-testid="new_password"
                      id="resetPassword_password"
                      type="password"
                      value=""
                    />
                    <span
                      class="ant-input-suffix"
                    >
                      <span
                        aria-label="eye-invisible"
                        class="anticon anticon-eye-invisible ant-input-password-icon"
                        role="img"
                        tabindex="-1"
                      >
                        <svg
                          aria-hidden="true"
                          data-icon="eye-invisible"
                          fill="currentColor"
                          focusable="false"
                          height="1em"
                          viewBox="64 64 896 896"
                          width="1em"
                        >
                          <path
                            d="M942.2 486.2Q889.47 375.11 816.7 305l-50.88 50.88C807.31 395.53 843.45 447.4 874.7 512 791.5 684.2 673.4 766 512 766q-72.67 0-133.87-22.38L323 798.75Q408 838 512 838q288.3 0 430.2-300.3a60.29 60.29 0 000-51.5zm-63.57-320.64L836 122.88a8 8 0 00-11.32 0L715.31 232.2Q624.86 186 512 186q-288.3 0-430.2 300.3a60.3 60.3 0 000 51.5q56.69 119.4 136.5 191.41L112.48 835a8 8 0 000 11.31L155.17 889a8 8 0 0011.31 0l712.15-712.12a8 8 0 000-11.32zM149.3 512C232.6 339.8 350.7 258 512 258c54.54 0 104.13 9.36 149.12 28.39l-70.3 70.3a176 176 0 00-238.13 238.13l-83.42 83.42C223.1 637.49 183.3 582.28 149.3 512zm246.7 0a112.11 112.11 0 01146.2-106.69L401.31 546.2A112 112 0 01396 512z"
                          />
                          <path
                            d="M508 624c-3.46 0-6.87-.16-10.25-.47l-52.82 52.82a176.09 176.09 0 00227.42-227.42l-52.82 52.82c.31 3.38.47 6.79.47 10.25a111.94 111.94 0 01-112 112z"
                          />
                        </svg>
                      </span>
                    </span>
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="ant-form-item css-dev-only-do-not-override-txh9fw"
        >
          <div
            class="ant-row ant-form-item-row css-dev-only-do-not-override-txh9fw"
          >
            <div
              class="ant-col ant-form-item-label ant-col-xs-24 ant-col-sm-12 css-dev-only-do-not-override-txh9fw"
            >
              <label
                class="ant-form-item-required"
                for="resetPassword_confirm"
                title="Confirm Password"
              >
                Confirm Password
              </label>
            </div>
            <div
              class="ant-col ant-form-item-control ant-col-xs-24 ant-col-sm-16 css-dev-only-do-not-override-txh9fw"
            >
              <div
                class="ant-form-item-control-input"
              >
                <div
                  class="ant-form-item-control-input-content"
                >
                  <span
                    class="ant-input-affix-wrapper ant-input-password ant-input-affix-wrapper-has-feedback css-dev-only-do-not-override-txh9fw"
                  >
                    <input
                      aria-required="true"
                      class="ant-input css-dev-only-do-not-override-txh9fw"
                      data-testid="confirm_password"
                      id="resetPassword_confirm"
                      type="password"
                      value=""
                    />
                    <span
                      class="ant-input-suffix"
                    >
                      <span
                        aria-label="eye-invisible"
                        class="anticon anticon-eye-invisible ant-input-password-icon"
                        role="img"
                        tabindex="-1"
                      >
                        <svg
                          aria-hidden="true"
                          data-icon="eye-invisible"
                          fill="currentColor"
                          focusable="false"
                          height="1em"
                          viewBox="64 64 896 896"
                          width="1em"
                        >
                          <path
                            d="M942.2 486.2Q889.47 375.11 816.7 305l-50.88 50.88C807.31 395.53 843.45 447.4 874.7 512 791.5 684.2 673.4 766 512 766q-72.67 0-133.87-22.38L323 798.75Q408 838 512 838q288.3 0 430.2-300.3a60.29 60.29 0 000-51.5zm-63.57-320.64L836 122.88a8 8 0 00-11.32 0L715.31 232.2Q624.86 186 512 186q-288.3 0-430.2 300.3a60.3 60.3 0 000 51.5q56.69 119.4 136.5 191.41L112.48 835a8 8 0 000 11.31L155.17 889a8 8 0 0011.31 0l712.15-712.12a8 8 0 000-11.32zM149.3 512C232.6 339.8 350.7 258 512 258c54.54 0 104.13 9.36 149.12 28.39l-70.3 70.3a176 176 0 00-238.13 238.13l-83.42 83.42C223.1 637.49 183.3 582.28 149.3 512zm246.7 0a112.11 112.11 0 01146.2-106.69L401.31 546.2A112 112 0 01396 512z"
                          />
                          <path
                            d="M508 624c-3.46 0-6.87-.16-10.25-.47l-52.82 52.82a176.09 176.09 0 00227.42-227.42l-52.82 52.82c.31 3.38.47 6.79.47 10.25a111.94 111.94 0 01-112 112z"
                          />
                        </svg>
                      </span>
                    </span>
                  </span>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="ant-divider css-dev-only-do-not-override-txh9fw ant-divider-horizontal"
          role="separator"
          style="margin-bottom: 10px;"
        />
        <div
          style="text-align: center;"
        >
          <button
            class="ant-btn css-dev-only-do-not-override-txh9fw ant-btn-primary"
            data-testid="submit_button"
            type="submit"
          >
            <span>
              Submit
            </span>
          </button>
        </div>
      </form>
    </div>
  </div>
</div>
`;
